.github/workflows/ci-workflow.yml

```diff
@@ -79,6 +79,10 @@ jobs:
     - name: Run Versioned Tests (npm v6 / Node 10-14)
       if: ${{ matrix.node-version != '16.x' }}
       run: npm run versioned:npm6
+      env:
+        VERSIONED_MODE: ${{ github.ref == 'main' && '--minor' || '--major' }}
```

Comment:
this basically restores the way it _used_ to run in travis.  so if we're on main run with `--minor` flag otherwise use `--major`

Reply:
👍  I like doing setting of the var directly in the workflow instead of CI specific in the runner script.